### PR TITLE
Release: Scene Fixes

### PR DESCRIPTION
**Pull request by mikedh**
- release #1564 
- release #1568 
- release #1574
- release #1575  
- release #1571

### CODE REVIEW

File: trimesh/points.py [hunk 1]
```diff
@@ -110,26 +111,33 @@ def radial_sort(points,
...
+        axis1 = np.cross(axis0, normal)
+    vectors = points - origin
     # calculate the angles of the points on the axis
-    angles = np.arctan2(pr0, pr1)
-
+    angles = np.arctan2(np.dot(vectors, axis0),
+                        np.dot(vectors, axis1))
     # return the points sorted by angle
-    return points[[np.argsort(angles)]]
+    return points[angles.argsort()[::-1]]
```
Comment (YuanWenqing) [inline trimesh/points.py:140]: Sorting points as `angles` descending is not corresponding to the doc "Vector to specify position of 12 in clock viewing in direction of normal".

Is this a typo?

Comment (mikedh) [reply trimesh/points.py:140]: Yeah I just changed that for the unit test so the result would match a theta `np.linspace(0, np.pi*2, 1000)` but don't have a strong opinion (increasing vs decreasing). Happy to match behavior of the docstring (descending theta) or change the docstring to "increasing" if you have a preference.

Comment (YuanWenqing) [reply trimesh/points.py:140]: Changing the docstring seems more reasonable: #1582

Comment (mikedh) [reply trimesh/points.py:141]: Thanks for the fix!

**Pull request merged**